plugins/azure-devops-backend/src/api/AzureDevOpsApi.ts

```diff
@@ -138,6 +142,103 @@ export class AzureDevOpsApi {
 
     return pullRequests;
   }
+
+  public async getBuildDefinitions(
+    projectName: string,
+    definitionName: string,
+  ): Promise<BuildDefinitionReference[]> {
+    this.logger?.debug(
+      `Calling Azure DevOps REST API, getting Build Definitions for ${definitionName} in Project ${projectName}`,
+    );
+
+    const client = await this.webApi.getBuildApi();
+    return client.getDefinitions(
+      projectName,
+      definitionName,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+    );
+  }
+
+  public async getBuilds(
+    projectName: string,
+    top: number,
+    repoId?: string,
+    definitions?: number[],
```

Comment:
Could the `definitions` parameter be added to the existing `getBuildList` method? It doesn't look like this method is doing much differently.

Reply:
I'm trying to avoid breaking changes